src/main/java/cubicchunks/server/CubeIO.java

```diff
@@ -363,7 +362,7 @@ private Cube readCubeFromNbtAndAddToColumn(Column column, final int cubeX, final
 		final Cube cube = column.getOrCreateCube(cubeY, false);
 
 		// get the worldgen stage
-		cube.setGeneratorStage(GeneratorStage.values()[nbt.getByte("GeneratorStage")]);
+		cube.setGeneratorStage(world.getGeneratorPipeline().getStage(nbt.getByte("GeneratorStage")));
```

Comment:
Right now breaking world format is OK. So it may be better to use a String here.